tasks.py

```diff
@@ -268,7 +262,57 @@ def build(_c, debug=False):
         image.tag(constants.IMAGE, tag=latest_tag)
 
 
-@task(pre=[lint, build])
+@task
+def sbom(_c, debug=False):
+    """Generate an SBOM"""
+    if debug:
+        getLogger().setLevel("DEBUG")
+
+    for variant in constants.VARIANTS:
+        versioned_tag = CONTEXT[variant]["buildargs"]["VERSION"]
+        image_and_tag = f"{constants.IMAGE}:{versioned_tag}"
+
+        try:
+            artifact_labels = utils.get_artifact_labels(variant=variant)
+
+            for iteration, label in enumerate(artifact_labels):
+                if iteration > 0:
+                    prior_label = artifact_labels[iteration - 1]
+                    prior_file_name = f"sbom.{prior_label}.json"
+                file_name = f"sbom.{label}.json"
+
+                if Path(file_name).is_file() and Path(file_name).stat().st_size > 0:
+                    LOG.info(f"Skipping {file_name} because it already exists...")
+                    continue
+
+                if iteration > 0:
+                    LOG.info(
+                        f"Copying {prior_file_name} into {file_name} since they are the same..."
+                    )
+                    shutil.copy(prior_file_name, file_name)
+                    continue
+
+                LOG.info(f"Generating {file_name} from {image_and_tag}...")
+                subprocess.run(
+                    [
+                        "syft",
+                        f"docker:{image_and_tag}",
+                        "-o",
+                        "json",
+                        "--file",
+                        file_name,
+                    ],
+                    capture_output=True,
+                    check=True,
+                )
+        except subprocess.CalledProcessError as error:
+            LOG.error(
+                f"stdout: {error.stdout.decode('utf-8')}, stderr: {error.stderr.decode('utf-8')}"
+            )
+            sys.exit(1)
+
+
+@task(pre=[build, sbom])
```

Comment:
This was `[lint, build]` and is now `[build, sbom]`.  `lint` isn't needed, because it happens in CI as a github action, and `sbom` is now needed because its output (the `sbom.*.json` files) need to be used as a part of the security tests.

Reply:
I was looking to see where the SBOM would be built. Good.